jupyter_book/book_template/_includes/metadata.json

```diff
@@ -1,16 +1,11 @@
 {
   "@context": "http://schema.org",
   "@type": "NewsArticle",
-  "mainEntityOfPage":
-    "{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}",
-  "headline":
-    "{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}",
-  "datePublished":
-    "{% if page.date %}{{ page.date | date_to_xmlschema }}{% else %}{{ site.time | date_to_xmlschema }}{% endif %}",
-  "dateModified":
-    "{% if page.date %}{{ page.date | date_to_xmlschema }}{% else %}{{ site.time | date_to_xmlschema }}{% endif %}",
-  "description":
-    "{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}",
```

Comment:
Sorry about the untidy diff here; I use VSCode and it apparently uses a JSON auto-formatter. Only the last line was actually edited.